internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -44,6 +47,9 @@ const (
 	imageName = "controller:latest"
 )
 
+// AnsibleOperatorVersion is the version of the base image and operator binary used in the Makefile
+var AnsibleOperatorVersion = strings.TrimSuffix(version.Version, "+git")
```

Comment:
This will no longer be the case soon, since ansible and helm operators are going into their own repos.

Reply:
But Ansible/Helm will also have a version.